spec/models/scraper_spec.rb

```diff
@@ -183,6 +183,22 @@
   context "there is a scraper" do
     let(:scraper) { Scraper.new }
 
+    context "scraper has no data" do
+      describe "#has_data?" do
+        it{expect(scraper.has_data?).to eq false}
+      end
+    end
+
+    context "scraper has data" do
+      before :each do
+        allow(scraper).to receive(:sqlite_db_size).and_return(256000)
```

Comment:
As far as I'm aware it's perfectly possible to have an sqlite database with a non-zero size that has no data in it. So, I don't think this is the best way to measure this